pkg/servicemodel/rc/v1/defs.go

```diff
@@ -10,3 +10,7 @@ const (
 	version       = "v1"
 	modelOID      = "1.3.6.1.4.1.53148.1.1.2.3"
 )
+
+const (
+	eventTriggerStyle2 = "Call Process Breakpoint"
```

Comment:
@eroshiva  do we have constants define for event trigger names in service model or these are not defined in the ASN.1?

Reply:
They are not defined in the ASN.1. This is something to be in specification, I guess